### PR TITLE
feat: history and playlist management

### PR DESCRIPTION
**Pull request by jalajcodes**
Added playlist management and history page.

https://max-video.herokuapp.com

### CODE REVIEW

File: frontend/src/reducer/sharedReducer.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const initialState = {
+  playlists: [],
+  history: [],
+  watchLater: [],
+  likes: [],
+  dislikes: [],
+};
+
+// {
+//     playlists: [{name: "", videos: []}, {...}],
```
Comment (vishal804) [inline frontend/src/reducer/sharedReducer.js:10]: Remove commented Code If not needed

**Review by vishal804 — APPROVED** [review]

**Comment by vishal804** [comment]
Code is readable And UI is Amazing Nice Work

File: frontend/src/reducer/sharedReducer.js (hunk 1 above)
Comment (jalajcodes) [reply frontend/src/reducer/sharedReducer.js:10]: actually I need that comment for now, will remove it in a future pr

**Review by vrinda-mahajan — APPROVED** [review]
Gone through the code. Found nothing to suggest. The code looks clean and readable.💯

**Pull request merged**